.gitignore

```diff
@@ -22,5 +22,5 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
-bundle.js
+bundle2.js
```

Comment:
bundle2?  did the name of the bundle generated by Webpack change?

Reply:
I am working on this!